src/conversion/modelconversiondispatcher.js

```diff
@@ -389,6 +396,14 @@ export default class ModelConversionDispatcher {
 		return consumable;
 	}
 
+	_createRangeConsumable( range ) {
+		const consumable = new Consumable();
+
+		consumable.add( range, 'range' );
```

Comment:
There should be an action to consume, not `range`, so for instance `makerCreate`, `makerRemove`.